depends/windows/angle/CMakeLists.txt

```diff
@@ -0,0 +1,372 @@
+add_library(angle::preprocessor ALIAS angle_preprocessor)
+
+##########
+# angle::translator
+add_library(angle_translator OBJECT ${angle_translator_sources}
+                                    ${angle_translator_essl_sources}
+                                    ${angle_translator_glsl_sources}
+                                    ${angle_translator_hlsl_sources})
+target_compile_definitions(angle_translator PUBLIC -DANGLE_ENABLE_ESSL
+                                                   -DANGLE_ENABLE_GLSL
+                                                   -DANGLE_ENABLE_HLSL)
+target_link_libraries(angle_translator PRIVATE angle::preprocessor)
+add_library(angle::translator ALIAS angle_translator)
+
+##########
+# libANGLE
+
+## OpenGL Renderer
+if((WINDOWS_DESKTOP AND ANGLE_ENABLE_WGL) OR LINUX OR APPLE)
+  add_library(angle_renderer_opengl INTERFACE)
+  if(WINDOWS_DESKTOP)
+    target_sources(angle_renderer_opengl INTERFACE $<BUILD_INTERFACE:${libangle_gl_sources}>
+                                                   $<BUILD_INTERFACE:${libangle_gl_wgl_sources}>)
+  elseif(APPLE)
+    target_sources(angle_renderer_opengl INTERFACE $<BUILD_INTERFACE:${libangle_gl_sources}>
+                                                   $<BUILD_INTERFACE:${libangle_gl_cgl_sources}>)
+  elseif(LINUX)
+    target_sources(angle_renderer_opengl INTERFACE $<BUILD_INTERFACE:${libangle_gl_sources}>
+                                                   $<BUILD_INTERFACE:${libangle_gl_glx_sources}>)
+  endif()
+
+  target_sources(angle_renderer_opengl INTERFACE $<BUILD_INTERFACE:${libangle_gl_sources}>
+                                                 $<BUILD_INTERFACE:${libangle_gl_wgl_sources}>)
+  target_compile_definitions(angle_renderer_opengl INTERFACE -DANGLE_ENABLE_OPENGL)
+  add_library(angle::renderer::opengl ALIAS angle_renderer_opengl)
+endif()
+
+# D3D Renderers
+if(WINDOWS_ANY)
+  ## WinRT D3D Renderer
+  if(WINDOWS_STORE)
+    add_library(angle_renderer_winrt INTERFACE)
+    target_sources(angle_renderer_winrt INTERFACE $<BUILD_INTERFACE:${libangle_d3d11_winrt_sources}>)
+    target_link_libraries(angle_renderer_winrt INTERFACE d3dcompiler.lib)
+    add_library(angle::renderer::winrt ALIAS angle_renderer_winrt)
+  endif()
+
+  ## Win32/d3d9 D3D Renderer
+  if(WINDOWS_DESKTOP AND ANGLE_ENABLE_D3D9)
+    add_library(angle_renderer_win32 INTERFACE)
+    target_sources(angle_renderer_win32 INTERFACE $<BUILD_INTERFACE:${libangle_d3d9_sources}>)
+    target_compile_definitions(angle_renderer_win32 INTERFACE -DANGLE_ENABLE_D3D9)
+    target_link_libraries(angle_renderer_win32 INTERFACE d3d9.lib)
+    add_library(angle::renderer::win32 ALIAS angle_renderer_win32)
+    list(APPEND ANGLE_DEPEND_LIBRARIES d3d9.lib)
+  endif()
+
+  ## D3D11 Base renderer
+  if(ANGLE_ENABLE_D3D11)
+    add_library(angle_renderer_d3d INTERFACE)
+    target_sources(angle_renderer_d3d INTERFACE $<BUILD_INTERFACE:${libangle_d3d_shared_sources}>
```

Comment:
as I already said. `libangle_d3d_shared_sources` are common for d3d11 and d3d9. if ANGLE_ENABLE_D3D11=OFF but ANGLE_ENABLE_D3D9=ON then build will fail.